### ISSUE TITLE
App/service to plot layered points/paths/regions on a map, particularly streets

### ISSUE THREAD

**Issue by johnbillion** [johnbillion/ideas#5]
I commonly read local news articles which mention street names, and particularly names of inter-sectioning streets ("on the corner of Church Lane and Market Street"), and I don't know the exact location.

I may know where Church Lane is but I don't know where it intersects with Market Street. This makes visualising the location of a news story very difficult.

I want an app or service which allows me to throw location names into it and plot them all on the same map so I can collect reference points. Saving would be neat but not necessary.

[OSM](https://www.openstreetmap.org/) allows you to plot a street (partially, at least) by selecting its node:

<img width="668" alt="" src="https://user-images.githubusercontent.com/208434/32201857-f83017de-bdd0-11e7-8bf8-bb09ae288bf7.png">

[uMap](http://umap.openstreetmap.fr/en/) is the closest I've seen but its search is poor and there doesn't appear to be a means of plotting a street given its name.

I actually built [an MVP](http://bit.ly/2gX963l) a few years ago using the Mapbox but it's not very reliable. Zooming breaks things. Search terms need to be accurate and specific.

<img width="353" alt="" src="https://user-images.githubusercontent.com/208434/32202507-35860b9e-bdd5-11e7-89c8-f46e5cd6213b.png">

[OpenLayers](http://openlayers.org/) might also be useful.

**Comment by uamv** [comment]
Have you utilized My Maps via Google which allows for self-defined layers or in what ways do you see this differing from that service?

**Comment by johnbillion** [comment]
Yeah I've used My Maps in the past but the plotting of streets is the missing feature there. OSM is so far the only maps platform I've found that provides this.

**Comment by uamv** [comment]
Ah, I see. You can kind of cheat that in Google … by finding a street, then adding a route from one end of the street to the other. But, a cheat.
